migrations/020.do.publish-rsr.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE unpublished_rsr_rounds (
+  round_index NUMERIC PRIMARY KEY,
+  evaluated_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  spark_evaluate_version TEXT NOT NULL,
+  measurement_commitments TEXT[] NOT NULL,
+  round_details JSONB NOT NULL,
+  providers JSONB NOT NULL
```

Comment:
I find the name `providers` as too ambiguous/potentially confusing, since this does not store a list of providers, but a list of per-provider retrieval stats.

The smart-contract uses the name `providerRetrievalResultStats`, which I find too long and redundant in the scope of `unpublished_rsr_rounds`.

How about this?

```suggestion
  providers_stats JSONB NOT NULL
```

Not a big deal, we can keep the current name if you prefer. It's an internal implementation detail that can be easily changed later.